pom.xml

```diff
@@ -44,7 +44,7 @@
 	</scm>
 
 	<properties>
-		<wicket.version>8.0.0-M3</wicket.version>
+		<wicket.version>8.0.0-SNAPSHOT</wicket.version>
```

Comment:
Is there a `<repository>` definition in this pom.xml for -SNAPSHOTs ?
If there is no such then the build at CI will fail!
Not everyone uses cutting edge like you, @solomax ! ;-)